.github/workflows/testing.yml

```diff
@@ -103,10 +104,18 @@ jobs:
           go version
           go test ${{ matrix.testflags }} -cpu 1,4 -timeout 7m google.golang.org/grpc/...
           cd "${GITHUB_WORKSPACE}"
+          set -x
```

Comment:
Remove?

Reply:
Why not just keep this in to enable debugging? I'll delete though.